MCEq/geometry/density_profiles.py

```diff
@@ -1058,31 +1058,25 @@ def latitude(self, det_zenith_deg):
         ) * np.cos(theta_rad)
 
         return (
-            -90.0
-            + np.arctan2(x * np.sin(theta_rad), r - d + x * np.cos(theta_rad))
+            np.arctan2(x * np.sin(theta_rad), r - d + x * np.cos(theta_rad))
```

Comment:
Thanks, for the PR. Why did you remove 90 degrees from latitude and still call it latitude?

Reply:
Fair point. I've changed the name of the function to avoid confusion.